### PR TITLE
Add lottery results section

### LINKED ISSUE #987: Listing: Lottery Results on Listing
As an applicant, 
I want to see when the public lottery will be
I also want to know when I should expect results
And then when results are posted
I want to be able to access them 

Notes
* There should be one lottery component with multiple states
* Public Lottery (listing can be open or closed) 
* Lottery Results Coming Soon (this is after lottery has been run, before results are posted, listing is closed at this point, we should trigger this state after the lottery day end 5pm) 
* Lottery Posted (we'll manually the lottery results PDF add which will trigger this state) 

**Public Lottery**
image

**Lottery Results Coming Soon**
image

**Lottery Posted**
image

### PR DESCRIPTION
**Pull request by software-project**
Resolves #987 

Note: I'm wondering if this is a good time to extract event components... either to just theirs own file or to ui-components @slowbot @jaredcwhite @dominikx96 what do you think?

### CODE REVIEW

**Comment by software-project** [comment]
@jaredcwhite I went ahead and extracted those components to ui-components. Could you have a look if it's fine?

**Comment by jaredcwhite** [comment]
Great idea! I think I'll redo my work in #1045 to go off of this instead.

File: sites/public/pages/listing.tsx [hunk 1]
```diff
@@ -326,13 +302,29 @@ export default class extends Component<ListingProps> {
               <aside className="w-full static md:absolute md:right-0 md:w-1/3 md:top-0 sm:w-2/3 md:ml-2 h-full md:border border-gray-400 bg-white">
                 <div className="hidden md:block">
                   <ApplicationStatus listing={listing} />
-                  {openHouseEvents && <OpenHouseEventSection openHouseEvents={openHouseEvents} />}
+                  {lotteryResults && lotteryResults.url && (
+                    <section className="aside-block -mx-4 pt-0 md:mx-0 md:pt-4 text-center">
```
Comment (jaredcwhite) [inline sites/public/pages/listing.tsx:306]: Is this also something we should extract out to a component?

Comment (software-project) [reply sites/public/pages/listing.tsx:306]: uhhh, yeah missed that

**Comment by software-project** [comment]
@slowbot do you think like `listing_sidebar/events` would be a better place for it?

**Pull request merged**